apps/epic-web/src/inngest/functions/sanity/product/index.ts

```diff
@@ -32,7 +32,11 @@ export const loadSanityProduct = async (
           },
           image{
             url
-          }
+          }, 
+           "instructor": contributors[0].contributor->{
+            userId,
+            name,    
```

Comment:
I'm not a huge fan of this and we should probably not hard code a zero-index lookup here and consider multiple instructors immediately.